frontend/test/services/GooglePlacesService.test.ts

```diff
@@ -0,0 +1,120 @@
+import { AxiosError } from "axios";
+import { GooglePlacesService } from "../../src/services/GooglePlacesService";
+import * as dotenv from "dotenv";
+
+/* eslint-disable  @typescript-eslint/no-unused-vars */
+// todo: remove eslint-disable once we add assertions
+describe("Unit tests for GooglePlacesService", function () {
+  const googlePlacesAPI = new GooglePlacesService();
+
+  beforeEach(function () {
+    dotenv.config();
+  });
+
+  test("Search for a place using restaurant name, should be successful", () => {
+    return googlePlacesAPI
+      .getRestaurantPlaceID("Jinya")
+      .then((result: string) => {
+        expect(result).toEqual("ChIJ4dRcUH5xhlQREcvYYxzhqv0");
+      })
+      .catch((error: AxiosError) => {
+        fail("Did not expect to fail: " + error.message);
+      });
+  });
+
+  test("Get details for a place using placeID, should be successful", () => {
+    // placeID can only be retrieved from getGooglePlacesSearch
+    return googlePlacesAPI
+      .getRestaurantDetails("ChIJb0n5cWl3hlQRIbVGYLiTEgE")
+      .then((result: Record<string, unknown>) => {
+        expect(result).toHaveProperty("business_status");
+        expect(result).toHaveProperty("formatted_address");
```

Comment:
Will we also store lat lon in the restaurant details?

Reply:
Yes! It's currently in the geometry field but I like your interface idea so I might do that and extract those details out from geometry

Reply:
Sounds good!